README.md

```diff
@@ -42,7 +42,8 @@ Example of possible config.
       [{handler, ergw_aaa_diameter},
        {host, <<"ergw-pgw.dia.example.net">>},
        {realm, <<"dia.example.net">>},
-       {connect_to, <<"aaa://srv1.dia.example.net;transport=sctp">>}
+       {connect_to, <<"aaa://srv1.dia.example.net;transport=sctp">>},
```

Comment:
In an existing config file (products/rpg) there is an additional level, {transports, ...}, before {connect_to, ...}. Like this
          [{handler, ergw_aaa_diameter},
           {'Origin-Host', <<"{{ .Values.hostnameRf }}">>},
           {'Origin-Realm', <<"{{ .Values.realmRf }}">>},
           {transports,
            [
        [{connect_to, <<"aaa://{{ .Values.cdfNode }};transport=sctp">>}]
           ]
         }
        ]},
Will that be a problem for this change?

Reply:
It rather looks like the sample in the README is different from reality.